### PR TITLE
Add new field to old publisher.

### PR DESCRIPTION
**Pull request by dianakhuang**
😿 😭 
https://openedx.atlassian.net/browse/WEBSITE-37

### CODE REVIEW

File: course_discovery/apps/publisher/templates/publisher/course_detail.html [hunk 1]
```diff
@@ -70,7 +70,13 @@
                 <div class="current number">{{ object.number }}</div>
                 <div class="show-diff"></div>
             </div>
-
+            <div class="info-item">
+                <div class="heading course-image-heading">
+                    {% trans "Course URL" %}
+                </div>
+                <div class="current number">{{ object.url_slug }}</div>
```
Comment (rgraber) [inline course_discovery/apps/publisher/templates/publisher/course_detail.html]: copy pasta on class

File: course_discovery/apps/publisher/templates/publisher/course_detail.html [hunk 2]
```diff
@@ -70,7 +70,13 @@
                 <div class="current number">{{ object.number }}</div>
                 <div class="show-diff"></div>
             </div>
-
+            <div class="info-item">
+                <div class="heading course-image-heading">
```
Comment (rgraber) [inline course_discovery/apps/publisher/templates/publisher/course_detail.html]: why the additional heading class?

File: course_discovery/apps/publisher/templates/publisher/course_edit_form.html [hunk 3]
```diff
@@ -157,6 +157,47 @@ <h1 class="hd-1 emphasized">{% trans "Edit Course" %}</h1>
...
+                                                    {% trans "Best Practices" %}
+                                                </li>
+                                                <li class="course-tabs" data-tab="tab-example">
+                                                    {% trans "Examples" %}
+                                                </li>
+                                            </ul>
+                                        </div>
+                                        <div id="tab-practices" class="content active">
+                                            <ul>
+                                                <li>{% trans "The default URL slug is based on the course title. This URL slug can be changed, and learners who visit an older URL will be redirected to the current URL." %}</li>
```
Comment (rgraber) [inline course_discovery/apps/publisher/templates/publisher/course_edit_form.html:175]: can you add in a "not currently used" line

**Review by rgraber — APPROVED** [review]
Tiny comment about adding a header, otherwise looks good!

File: course_discovery/apps/publisher/templates/publisher/course_edit_form.html (hunk 3 above)
Comment (dianakhuang) [reply course_discovery/apps/publisher/templates/publisher/course_edit_form.html:175]: There is one up here, but I'm happy to add another one if necessary: https://github.com/edx/course-discovery/pull/2085/files#diff-d356051f351a1439b9bb6e6ac1c1f0a5R160

**Pull request merged**